Storage/StorageTests/CoreData/MigrationTests.swift

```diff
@@ -1498,14 +1517,29 @@ private extension MigrationTests {
             "shippingPostcode": "",
             "shippingState": ""
         ])
+
+        // Required since model 75
+        if modelVersion >= 75 {
+            customer.setValue(1, forKey: "siteID")
+        }
+
+        return customer
     }
 
     /// Inserts a `CustomerSearchResult` entity, providing default values for the required properties.
     @discardableResult
-    func insertCustomerSearchResult(to context: NSManagedObjectContext) -> NSManagedObject {
-        context.insert(entityName: "CustomerSearchResult", properties: [
+    func insertCustomerSearchResult(to context: NSManagedObjectContext, forModel modelVersion: Int) -> NSManagedObject {
+        let customerSearchResult = context.insert(entityName: "CustomerSearchResult", properties: [
             "customerID": 1
         ])
+
+        // Required since model 75
+        if modelVersion >= 75 {
+            customerSearchResult.setValue(1, forKey: "siteID")
+            customerSearchResult.setValue("", forKey: "keyword")
+        }
+
+        return customerSearchResult
```

Comment:
Here I forgot to remove the `customerID` in the new model (75), as doesn't exist anymore, but changing the logic didn't resolve the problem of the `migrate()` method failing. For reference I changed it to:

```
    /// Inserts a `CustomerSearchResult` entity, providing default values for the required properties.
    @discardableResult
    func insertCustomerSearchResult(to context: NSManagedObjectContext, forModel modelVersion: Int) -> NSManagedObject {
        if modelVersion >= 75 {
            let customerSearchResult = context.insert(entityName: "CustomerSearchResult", properties: [
                "siteID": 1,
                "keyword": "hi"
            ])
            return customerSearchResult
        } else {
            let customerSearchResult = context.insert(entityName: "CustomerSearchResult", properties: [
                "customerID": 1
            ])
            return customerSearchResult
        }
    }
```